client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -353,7 +353,7 @@ public <K, V, C> ShuffleHandle registerShuffle(
 
     shuffleIdToPartitionNum.putIfAbsent(shuffleId, dependency.partitioner().numPartitions());
     shuffleIdToNumMapTasks.putIfAbsent(shuffleId, dependency.rdd().partitions().length);
-    if (shuffleManagerRpcServiceEnabled && rssStageRetryEnabled) {
+    if (shuffleManagerRpcServiceEnabled && rssStageRetryForWriteFailureEnabled) {
```

Comment:
Why the scope of stage retry reduced to the write failure ？

Reply:
> Why the scope of stage retry reduced to the write failure ？

Have been modified.